### PR TITLE
Create Github page

### PR DESCRIPTION
**Pull request by PauloGoncalvesBH**
Created a simple Github page to the protractor-helper.

 See a example here: https://paulogoncalvesbh.github.io/protractor-helper/

 If you approve that, you need to change settings.
 Go to 'Settings' > 'Options' > and choice the 'master branch' option on 'source / GitHub Pages'.

image

 That PR resolve the issue #23.

## Pull Request Checklist
	
	- [x] Have you followed the steps in our [Contributing](https://github.com/wlsf82/protractor-helper/blob/master/docs/CONTRIBUTING.md) document?
	- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wlsf82/protractor-helper/pulls) for the same update/change?
	- [ ] Have you successfully ran tests with your changes locally?
	- [ ] Have you written new tests for your changes, if necessary?
	- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
	- [ ] Have you made improvements to the documentation, if necessary? ([docs/](https://github.com/wlsf82/protractor-helper/tree/master/docs) folder and [README.md](https://github.com/wlsf82/protractor-helper/blob/master/README.md))
	- [ ] Have you linked the correct [GitHub issue](https://github.com/wlsf82/protractor-helper/issues), if there is any?
	
	> Pull requests that do not pass the automated tests on [SemaphoreCI](http://semaphoreci.com) and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed

### CODE REVIEW

**Comment by wlsf82** [comment]
:Wow: Super!

File: index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Protractor helper's library
+
+[![npm version](https://badge.fury.io/js/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) [![npm weekly downloads](https://img.shields.io/npm/dw/protractor-helper.svg)](https://www.npmjs.com/package/protractor-helper) 
+
+This library contains helper functions that can be used together with Protractor for creating robust end-to-end tests.
+
+Many of the helper functions on this library uses `protractor.ExpectedConditions` to ensure that the elements we want to interact with are in the correct state before interacting with them or running expectations on them. This helps on avoiding trying to interact with elements when they are still not ready for it, which helps on avoiding test flakiness.
```
Comment (wlsf82) [inline index.md]: Grammar fixes.
What about:
`Many of the helper functions on this library use `protractor.ExpectedConditions` to ensure that the elements we want to interact with are in the correct state before interacting with them or running expectations on them. This helps in avoiding trying to interact with elements when they are not ready for it, which helps in preventing test flakiness.`

File: index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Installation
+
+Below it is described the process of Installation of such module.
```
Comment (wlsf82) [inline index.md]: Shorten sentence.
What about:
`Below is described the library's installation process.`

File: index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## How to use and examples
+
+After installing the library you will need to require it in your test file (see below).
+
+```js
+const protractorHelper = require("protractor-helper");
+```
+
+As soon as you have the library required in your test file you can start using its helper methods.
```
Comment (wlsf82) [inline index.md]: Let's change it to:
`As soon as you have the library required in your test file you can start using its helper functions.`

File: index.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+[Here you'll find examples of usage of each of the available helper methods.](https://www.npmjs.com/package/protractor-helper#available-helpers)
```
Comment (wlsf82) [inline index.md]: Let's change it to:
`[Here you'll find examples of usage of each of the available helper functions.](https://www.npmjs.com/package/protractor-helper#available-helpers)`

Comment (PauloGoncalvesBH) [reply index.md]: Done.

File: index.md (hunk 3 above)
Comment (PauloGoncalvesBH) [reply index.md]: Done.

File: index.md (hunk 2 above)
Comment (PauloGoncalvesBH) [reply index.md]: Done.

File: index.md (hunk 1 above)
Comment (PauloGoncalvesBH) [reply index.md]: Done.

**Pull request merged**